templates/etcd-ha/3/README.md

```diff
@@ -1,41 +1,38 @@
 # Etcd 
 
-A distributed, highly-available key/value store written in Go.
+A distributed, highly-available key/value store for storing configuration information.
```

Comment:
It can store more then config info. Maybe something more like:
etcd is a distributed key value store that provides a reliable way to store data across a cluster of machines.

which is from coreOS's description.